config/nginx.conf

```diff
@@ -161,14 +194,43 @@ http {
                         ##### END OF PLUGIN EXTRA NGINX CONF (DYNAMIC) FOR APP {{APP.name}} #####
                     {% endif %}
 
-                    {% if APP.static_routing %}
-                        location /static/ {
+                    {% if APP.type != "static" %}
+                        {% if APP.static_routing %}
+                            location /static/ {
+                                {% if APP.extra_nginx_conf_static_string %}
+                                    ##### BEGIN OF PLUGIN EXTRA NGINX CONF (STATIC) FOR APP {{APP.name}} #####
+                                    {{APP.extra_nginx_conf_static_string}}
+                                    ##### END OF PLUGIN EXTRA NGINX CONF (STATIC) FOR APP {{APP.name}} #####
+                                {% endif %}
+                                alias {{APP.workdir}}/static/;
+                                {% if MFSERV_ADMIN_HOSTNAME != "null" %}
+                                    log_by_lua_block {
+                                        local stats = require("stats")
+                                        stats.send_status_code_stat("request_counter", "{{PLUGIN.name}}", "{{APP.app_name}}", "static")
+                                        stats.send_timing_stat("request_timer", "{{PLUGIN.name}}", "{{APP.app_name}}", "static")
+                                    }
+                                {% endif %}
+                            }
+                        {% endif %}
```

Comment:
the bloc in `{% if APP.static_routing %}` must be deleted because static_routing option does not make sense for static plugin (it's even not defined in config.ini)

Reply:
I don't understand : this bloc is only present for non static plugin...

Reply:
my bad :-)